charts/nri-kube-events/README.md

```diff
@@ -1,57 +1,149 @@
-| `podSecurityContext.runAsUser`                             | runAsUser UID for Pod Security Context.                                                                                                                                                                                                        | `1001`                          |
-| `podSecurityContext.runAsGroup`                            | runAsGroup GID for Pod Security Context.                                                                                                                                                                                                       | true                            |
-| `proxy`                                                    | Set proxy if you required as Kube-events-forwarder / infra-agent needs to send metrics to New Relic One.                                                                                                                                       |                                 |
-| `config`                                                   | Set config option in order to pass any configuration to the agent running as a sidecar. This object is map as a configuration map mounted in /etc/newrelic-infra.yml.                                                                          |                                 |
-| `deployment.annotations`                                   | The annotations to add to the `Deployment`.                                                                                                                                                                                                    | `{}`                            |
-| `customAttributes.*`                                       | Custom attributes/Extra Label that will be added to all the metrics sent by Kube events. Queryable in New Relic.                                                                                                                               | `{}`                            |
-
-## Example
-
-Make sure you have [added the New Relic chart repository.](../../README.md#installing-charts)
-
-Then, to install this chart, run the following command:
-
-```sh
-helm install newrelic/nri-kube-events \
---set licenseKey=<enter_new_relic_license_key> \
---set cluster=my-k8s-cluster
+![Version: 2.2.2](https://img.shields.io/badge/Version-2.2.2-informational?style=flat-square) ![AppVersion: 1.8.0](https://img.shields.io/badge/AppVersion-1.8.0-informational?style=flat-square)
+
+A Helm chart to deploy the New Relic Kube Events router
+
+**Homepage:** <https://docs.newrelic.com/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration>
+
+## Source Code
+
+* <https://github.com/newrelic/nri-kube-events/>
```

Comment:
maybe it makes sense to add the agent repo as well

Reply:
So it has to be added here: https://github.com/newrelic/nri-kube-events/pull/112/files/e1e03875036017eb0aa6f1dc159b5447165fd9e2#diff-fbf163ddbbb7a2d5bc0141b6395932787c29747d9eb25a2347104448534b0001R6-R8

I will add it in the next commit.

Reply:
Solved in [#9f9cfc0](https://github.com/newrelic/nri-kube-events/pull/112/commits/9f9cfc0e51242f18647739df110dcb9b8600a133)